Error in installation commands in README.md

# Expected Behavior

In the install section of README.md, the command for starting the application is supposed to be `npm run start:dev`

# Current Behavior

By using `npm start`, we get the error `npm ERR! missing script: start`

Can I make a PR fixing this issue?

--- a/README.md
+++ b/README.md
@@ -30,7 +30,7 @@ $ npm install
 $ It is recommended to install [MongoDB Compass](https://www.mongodb.com/try/download/compass) to visualize data in mongodb
 
 # Start application
-$ npm start
+$ npm run start:dev
 
 # Visit http://localhost:3030/ in your browser
 ```

